docs/examples/15_minutes_to_QCoDeS.ipynb

```diff
@@ -1111,7 +1124,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We refer reader to exporting data section of the [example notebook](http://qcodes.github.io/Qcodes/examples/DataSet/Dataset%20Context%20Manager.html?highlight=get_parameter_data) for further information on `get_parameter_data` method."
+    "We refer reader to exporting data section of the [Dataset Context Manager notebook](http://qcodes.github.io/Qcodes/examples/DataSet/Dataset%20Context%20Manager.html?highlight=get_parameter_data) and [Accessing data in DataSet notebook](DataSet/Accessing-data-in-DataSet.ipynb) for further information on `get_parameter_data` method."
```

Comment:
Should we also fix the other link here?

Reply:
@GateBuilder Just looked at this and while this link (with spaces ) resolves in the rendered html it does not resolve in the notebook server. I suggest either renaming the notebook or adding `%20` escapes for the spaces